shiksha-website/shiksha-frontend/src/app/shared/components/upload-popup/upload-popup.component.html

```
@@ -35,9 +35,9 @@ <h2 id="upload-dialog-title" class="text-lg font-bold mb-4">{{'Bulk Upload' | tr
             {{'Browse Files' | translate}}
           </button>
           <p class="text-content-100 text-sm">
-            {{'File supported .xlsx file, up to 5MB in size' | translate}}.
+            {{ fileHint | translate }}.
           </p>
-          <button class="text-nowrap text-base" (click)="downloadTemplate()">{{'Download Template' | translate}}</button>
+          <button *ngIf="showDownloadTemplate" class="text-nowrap text-base" (click)="downloadTemplate()">{{'Download Template' | translate}}</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)